src/components/Card/Card.jsx

```diff
@@ -6,21 +6,21 @@ import { commerce } from '../../lib/commerce';
 
 
 
-const Card = ({products}) => {
+const Card = ({item}) => {
 
  
 return (
-   <Link to={'/product/${products.id}'}>
+   <Link to={'/product/${item.id}'}>
     <div className='card max-w-[90%] max-h-2xl rounded-xl border-slate-200 border-4 overflow-hidden shadow-2xl hover:scale-110 mx-auto'>
       <div className='img-container aspect-[3/2] object-contain'>
         
-      <img src={products.image} alt='/' className='mainImg  w-[100%] max-h-80 object-cover ' />
+      <img src={item.image.url} alt='/' className='mainImg  w-[100%] max-h-80 object-cover ' />
```

Comment:
A lot of the fields in a single item that look like they should be "one thing" are actually an object.  So item.image is an object containing several properties, so is item.price, etc.